### ISSUE TITLE
Please, block ch1ld p0rn from the site.

### ISSUE THREAD

**Issue by ConcernedCitizen33** [btdig/dhtcrawler2#28]
I was in btdig.com looking for an adult performer and after a few pages, the algoritm suggested some links with the description of "child porn". Brothers, that is disgusting and a fellony, no matter the nationality or country that you live in. It seems tha the site contains a lot of links to archives like that. 
I urge and beg you guys to not tolarate that kind of content in the site. So, please, I ask the admins to search for it and delete or block that kind of content from the site.

**Comment by ghost** [comment]
This type of content is littered in every single corner of the internet. Use the god-given skill of ignoring things and let the people downloading such content get in trouble for doing so, natural selection they call it.

If Twitter, Instagram, and Facebook cannot effectively combat the spread of illegal content such as this, what makes you think the site operator is going to do a better job at it? Besides, the website has filters in place to prevent people from being able to search for such content after all, so if magnet URLs can be completely redacted, that would be censorship and I'm sure people 90% of people who torrent would be completely against such a thing.

**Comment by UglyBoiDesu** [comment]
It seems they have a poorly implemented method of combating CSAM #40 
It would be better to have a list of keywords to blacklist when showing up in results.

**Comment by luckyycode** [comment]
it's not just some links with ad op mentioned. I was just looking for a music album and what I had is 80 mp4 files with that content, very much appreciated, but I guess it's not indexer fault..

anyway, be careful! now I got to wipe my disk to metal lol

**Comment by NerOfTheRings** [comment]
They have blocked child porn, thank god, if you saerch for it now you'll be redirected to fbi page

**Comment by ProphetZarquon** [comment]
I've been using btdigg & btdig almost every week for literally years, & I have never had any issue of porn or especially underage porn, outnumbering relevant results. Additionally, **btdig doesn't host content**; it only shows what content exists; it does not show the content itself. If the goal is to reduce CP, then use a tool like btdig as the valuable discovery resource it is, rather than complaining that it reports existence too accurately. 

Underage content is a tiny fraction of what's exchanged via torrent: If relevant results are being outnumbered by irrelevant, that's because the crawler is being flooded with bad-faith peers, not because the DHT needs a censor on it.

**Comment by milahu** [comment]
> let the people downloading such content get in trouble for doing so

fun fact:
[german police raided my home and stole all my hardware](https://github.com/milahu/alchi/blob/master/deutsch/feedback/knast-2024/2024-12-09.beschluss-hausdurchsuchung-kinderpornos.md)
because i \*accidentally\* downloaded childporn
by searching for "small tits" on uncensored DHT search engines

i never saw the video, because i removed the torrent at 1% progress or so
the alleged childporn torrent hash is [589a84b8a4e297fc25bf4be9602d5ef11e4363dc](https://btdig.com/589a84b8a4e297fc25bf4be9602d5ef11e4363dc)
(is this an [illegal number](https://en.wikipedia.org/wiki/Illegal_number)?)
the display name is "媲美欣系列" = "Comparable to the Xin Series"
[i told the court that \*accidental\* downloading of childporn is legal](https://github.com/milahu/alchi/blob/master/deutsch/feedback/knast-2024/2025-02-01.kinderpornos-einlassung.md)
but they said "fuck off, we dont care, we keep your hardware for evaluation"
(i guess this "evaluation" will take 2 years, to sabotage my work)

lessons learned:
always have backups, use pornhub to watch porn

**Comment by MikitaNoceda** [comment]
> They have blocked child porn, thank god, if you saerch for it now you'll be redirected to fbi page

nope. still there. with select keywords it is still rather easy to find.
